src/embind/embind.js

```diff
@@ -50,6 +50,9 @@ var LibraryEmbind = {
     // names. This lets the test suite know that.
     Module['NO_DYNAMIC_EXECUTION'] = true;
 #endif
+#if EMBIND_STD_STRING_UTF8_SUPPORT
```

Comment:
Maybe this is more of a question for @kripken, but does changing the -D options to emcc cause embind.js to get reprocessed? I forget how all of this works.

Reply:
`emcc -DFOO` will define `FOO` for C, but not as an emscripten compiler setting. You need `-s KEY=VALUE` for that (and to define and document it in `src/settings.js`).